google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -433,9 +434,10 @@ def create_connect_args(self, url):
             ),
             url.database,
         )
+        dist = pkg_resources.get_distribution("sqlalchemy-spanner")
         return (
             [match.group("instance"), match.group("database"), match.group("project")],
-            {},
+            {"user_agent": dist.project_name + "/" + dist.version},
```

Comment:
If you could also add a quick unit test for this, that would be great. Just to see if it's getting set correctly.